test/prototype_transforms_dispatcher_infos.py

```diff
@@ -416,4 +419,12 @@ def xfail_all_tests(*, reason, condition):
             skip_dispatch_feature,
         ],
     ),
+    DispatcherInfo(
+        F.convert_dtype,
+        kernels={
+            features.Image: F.convert_dtype_image_tensor,
+            features.Video: F.convert_dtype_video,
+        },
+        method_name="to_dtype",
```

Comment:
This is needed since the method on the features is called `to_dtype` whereas the dispatcher is called `convert_dtype`. This is on par with the other conversion methods we already have, e.g.

https://github.com/pytorch/vision/blob/0610b13ac4af3717f538454a9c6b1f441cb386f3/torchvision/prototype/features/_image.py#L113

https://github.com/pytorch/vision/blob/0610b13ac4af3717f538454a9c6b1f441cb386f3/torchvision/prototype/transforms/functional/_meta.py#L264

This was not needed before, since the other conversion methods don't have `DispatcherInfo`'s yet.